MovingBlocks/Terasology
# Add ping to server browser

### Contains

- Server browser now shows ping in addition to other server information. 

It didn't make sense to bind this to the UIText like the other fields so this information is retrieved from inside the JoinGameScreen class whenever the user selects a server or clicks refresh.

~~Player overlay [tab] now shows the ping of other players in multiplayer.~~ 

~~To implement this, I went with adding a PingComponent and a PingAuthoritySystem which constantly updates the PingComponents of all connected players in a multiplayer server. The overlay screen then displays this information. I was unable to test this since the system needs to be on the server to receive the onConnected event so I'm not entirely sure if this work~~
 
There seem to have been some auto-formatting changes by IntelliJ as well.
### How to Test

Open the Join Game screen. Verify that the ping is shown for each server.

![capture](https://cloud.githubusercontent.com/assets/1729493/21051489/78e1bea8-be5b-11e6-80a9-a1dbec17a292.PNG)

Do let me know what areas of the code can be improved on 🙂.
